### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KevinWorkman**
Realized I left some TODO comments in the readme. All of this is covered in the website, so I'm just removing them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,8 +56,6 @@ If you don't see a version number, then make sure Git is on your `PATH`.
 
 ## Step 4: Clone this repository
```
Comment (wgenemorgan) [inline README.md]: This section is empty now. Remove it or add link to appropriate page.

Comment (wgenemorgan) [inline README.md]: Line 64: another TODO that should be removed.

Comment (KevinWorkman) [reply README.md]: Do we want the public repo pointing to the private site?

Comment (KevinWorkman) [reply README.md]: I've added a sentence explaining to follow the instructions in the first project.

Comment (KevinWorkman) [reply README.md]: Good catch. Done.

**Comment by KevinWorkman** [comment]
Thanks!

**Pull request merged**